sql-plugin/src/main/scala/com/nvidia/spark/rapids/basicPhysicalOperators.scala

```diff
@@ -93,6 +93,13 @@ case class GpuProjectExec(
    child: SparkPlan
  ) extends UnaryExecNode with GpuExec {
 
+  // we do not record totalTime for this operator
```

Comment:
This smells like boilerplate that's going to be replicated quite a few times as this change ripples through operators.  Should this be moved to a `GpuExec` object method for reuse?

Reply:
Yes, one option would be to remove `totalTime` from the default set of metrics and explicitly add it as an `additionalMetric` into all the operators that currently use it.